script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -1093,7 +1119,7 @@ The man turns his back on us all of a sudden.
 [sha:346a6167654ff1845fc3b591a2ddd938fb9d83c0]{
 -- Page 877, Offset 7995.
 -- 何事もなかったようにホールまで歩いて行くと、そのまま悠然とエレベーターに乗って、このフロアから立ち去っていった。
-He walks to the elevators as if nothing had happened, boards one with the same air of composure, and leaves the floor.
+He walks over to the elevators as if nothing had happened, boards one with the same air of composure, and leaves the floor.
```

Comment:
I'm imagining Vlov waiting a full minute for the elevator arrive while awkwardly ignoring Shiki + Arc